Userland/Applications/Presenter/PresenterSettingsFooter.gml

```diff
@@ -0,0 +1,49 @@
+@GUI::Widget {
+    fill_with_background_color: true
+    layout: @GUI::VerticalBoxLayout {
+        margins: [10]
+        spacing: 5
+    }
+
+    @GUI::Label {
+        text: "These settings allow you to override the footer settings of all presentations."
+        text_alignment: "TopLeft"
+        // FIXME: Use dynamic sizing once supported.
```

Comment:
Dynamic sizing should work I think? But Frhun might know better.

Reply:
It does not work properly for labels, as their height with text wrapping is not computed correctly.